indico/modules/events/abstracts/controllers/management.py

```diff
@@ -98,13 +100,68 @@ def _process(self):
         return send_file(file_name, pdf.generate(), 'application/pdf')
 
 
-class RHAbstracts(RHManageAbstractsBase):
+class RHAbstractsCallScheduleBase(RHManageAbstractsBase):
```

Comment:
What looks better, `RHAbstractsCFASchedule` or `RHCFASchedule`? I would say the first.